### PR TITLE
Review

### PR DESCRIPTION
**Pull request by sntgopl**
-Created linters files.
-Added complete ReadMe file
-Corrected linter errors.
-Opened pull request for own review.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Survey form</title>
+    <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+<h1 id="title">Survey Form</h1>
```
Comment (sntgopl) [inline index.html:10]: Please create a parent <div> element such as <header> to get a more smooth and correct managment for the head title section.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+<p id="description">Please complete this survey</p>
+<form id="survey-form">
```
Comment (sntgopl) [inline index.html:12]: Please create a parent <div> element for the form section such as "<section>" to get a correct usage of it.

Comment (sntgopl) [inline index.html:12]: Please use a 2 space identation for a more correct reading of the code and getting it cleaner.

File: styles.css [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+}
+
+form {
+  background-color: rgba(138, 26, 236, 0.5);
+  margin: auto;
+  text-align: left;
+  width: max-content;
+}
+
+#comment-area {
```
Comment (sntgopl) [inline styles.css:26]: Please use a class identification for this element for further use.

File: styles.css [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+  min-width: 500px;
+  max-width: 500px;
+}
+
+#title {
```
Comment (sntgopl) [inline styles.css:31]: Please use class identification instead of id for further use.

File: styles.css [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+  margin-top: 20px;
+  margin-bottom: 20px;
+}
+
+label {
+  margin-top: 20px;
+  margin-bottom: 20px;
+}
+
+#submit {
```
Comment (sntgopl) [inline styles.css:50]: Please use class identification for this element for further use.

File: styles.css [hunk 6]
```diff
@@ -0,0 +1,55 @@
...
+  padding: 25px;
+}
+
+input {
+  margin-top: 20px;
+  margin-bottom: 20px;
+}
+
+label {
+  margin-top: 20px;
```
Comment (sntgopl) [inline styles.css:46]: You can use shortcut usage for margin attribute such as "margin: top right bottom left;"

**Review by sntgopl — COMMENTED** [review]
Please go through comments made in the code for correct implementation of HTML & CSS best practice to get a cleaner code for yourself and pair-coders. Feel free to tag me in any comment you may need to request any help or review further on. 
Thank you.

**Review by leonardodiasb — APPROVED** [review]
Hi @sntgopl ,

Everything is looking great! Please, merge this branch :shipit:
Congratulations! 🎉

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**